src/mixins/render-vega-lite/Definitions/Transform/Transforms/CrossSection2dDefinitionObject.js

```diff
@@ -73,12 +73,16 @@ export default class CrossSection2dDefinitionObject extends PropertiesDefinition
       obj.crossSectionLine.length !== 2 ||
       !Array.isArray(obj.crossSectionLine[0]) ||
       obj.crossSectionLine[0].length !== 2 ||
-      typeof obj.crossSectionLine[0][0] !== "number" ||
-      typeof obj.crossSectionLine[0][1] !== "number" ||
+      (typeof obj.crossSectionLine[0][0] !== "number" &&
+        typeof obj.crossSectionLine[0][0] !== "string") ||
+      (typeof obj.crossSectionLine[0][1] !== "number" &&
+        typeof obj.crossSectionLine[0][1] !== "string") ||
       !Array.isArray(obj.crossSectionLine[1]) ||
       obj.crossSectionLine[1].length !== 2 ||
-      typeof obj.crossSectionLine[1][0] !== "number" ||
-      typeof obj.crossSectionLine[1][1] !== "number"
+      (typeof obj.crossSectionLine[1][0] !== "number" &&
+        typeof obj.crossSectionLine[1][0] !== "string") ||
+      (typeof obj.crossSectionLine[1][1] !== "number" &&
+        typeof obj.crossSectionLine[1][1] !== "string")
```

Comment:
This condition is so long it may be extra useful to make these shorter, not necessary but possible!

`!obj.crossSectionLine[0].every(p => ["number", "string"].includes(typeof p))`
`!obj.crossSectionLine[1].every(p => ["number", "string"].includes(typeof p))`

I think that comes out to the same... If everything is _not_ a number or a string it will throw the error

Reply:
golf:
`!obj.crossSectionLine.every(coord => coord.every(p => ["number", "string"].includes(typeof p)))